Sources/Bootstrap/Tags/Alert.swift

```diff
@@ -58,7 +49,14 @@ public final class AlertTag: TagRenderer {
 
         return tag.serializer.serialize(ast: body).map(to: TemplateData.self) { er in
             let body = String(data: er.data, encoding: .utf8) ?? ""
-            let alert = "<div class='alert alert-\(parsedStyle) \(classes ?? "")\" \(attributes ?? "")' role='alert'>\(body)</div>"
+
+            var alert = "<div class=\"alert alert-\(parsedStyle)"
+            if let classes = classes {
```

Comment:
It seems the previous code did not produce the right result (it would group attributes together with the classes).

A correct (?) one-liner (if you prefer) without double spaces could be:
```swift
let alert = "<div class='alert alert-\(parsedStyle)\(classes.map { " \($0)" } ?? "")'\(attributes.map { " \($0)" } ?? "") role='alert'>\(body)</div>"
```

or a bit more readable and with shorter lines:
```swift
let classesHTML = classes.map { " \($0)" } ?? ""
let attributesHTML = attributes.map { " \($0)" } ?? ""
let alert = "<div class='alert alert-\(parsedStyle)\(classesHTML)'\(attributesHTML) role='alert'>\(body)</div>"
```

Not sure which you prefer, I try to prevent `var`s when it's possible.

Reply:
First it was to fix it being class using a single quote then double to close it... caused a problem.. but yes it was being anal with spacing 😬